docs/evaluate/development-production-features/temporal-nexus.mdx

```diff
@@ -17,32 +17,96 @@ keywords:
+- **Multi-region use cases** \-
+Nexus requests in Temporal Cloud are routed across a global mTLS-secured Envoy mesh.
+Reliable execution is provided with built-in Nexus Machinery and Temporal's queue-based Worker architecture.
 
-Temporal Nexus provides an integrated Temporal SDK experience, built-in Nexus Machinery, first-class observability, and enables each team to have their own Namespace for improved security, troubleshooting, and blast radius isolation.
-It allows multiple teams to collaborate autonomously while maintaining a clean separation of concerns, ensuring that applications are scalable, secure, and easy to manage as they grow.
+**Advantages of Temporal Nexus:**
 
-With Temporal Nexus, teams are able to:
+- Reliably connect Temporal Applications within and across Namespaces.
+- Service contracts suitable for sharing across teams.
+- Abstract underlying Temporal primitives, like Workflows – or execute arbitrary code.
+- Modular design that streamlines multi-team development.
+- Improved security and blast-radius isolation with isolated Namespaces.
+- Same queue-based Worker architecture, no bespoke service deployments needed.
 
-- **Easily scale and manage applications** by breaking them into modular components developed separately while maintaining smooth integration through clear service contracts.
-- **Run Workflows across teams and Namespaces**, enabling durable execution and custom code to operate seamlessly across different parts of the system.
-- **Enhance security by isolating different parts of the application**, reducing the risk and impact of potential issues.
-- **Continue using familiar tools like queue-based Workers**, without the need for complex new setups or deployments.
+<RelatedReadContainer>
+  <RelatedReadItem path="https://youtu.be/qqc2vsv1mrU?feature=shared&t=2082" text="Nexus keynote & demo - Replay video" archetype="replay-talk" />
+  <RelatedReadItem path="https://www.youtube.com/watch?v=izR9dQ_eIe4" text="Nexus deep dive - Replay video" archetype="replay-talk" />
+  <RelatedReadItem path="/cloud/nexus" text="Production deployment in Temporal Cloud" archetype="feature-guide" />
+  <RelatedReadItem path="/production-deployment/self-hosted-guide/nexus" text="Self-hosted configuration to enable Nexus" archetype="feature-guide" />
+  <RelatedReadItem path="/develop/go/nexus" text="Go SDK Nexus - quick start and code sample" archetype="feature-guide" />
+  <RelatedReadItem path="/develop/java/nexus" text="Java SDK Nexus - quick start and code sample" archetype="feature-guide" />
+  <RelatedReadItem path="/nexus" text="Nexus" archetype="encyclopedia" />
+</RelatedReadContainer>
 
-_*Cross cloud calls are not available yet on Temporal Cloud. Only single cluster deployments are supported for self-hosted._
+_*The Nexus Registry is Account-scoped in Temporal Cloud and Cluster-scoped in self-hosted deployments._
```

Comment:
Consider using an admonition instead of an asterisk

Reply:
fixed and moved to the encyclopedia page